tiledbimg/openslide.py

```diff
@@ -50,17 +48,27 @@ def level_count(self) -> int:
         return len(self.level_info)
 
     @property
-    def dimensions(self) -> Tuple[int, ...]:
+    def dimensions(self) -> Tuple[int, int]:
         """A (width, height) tuple for level 0 of the slide."""
-        return self.level_info[0].dimensions[:2]
+        return self.level_info[0].dimensions
```

Comment:
I assume the changes `dimensions[:2]` is to match the function description `(width, height)`  and in future PRs we will generalise again?

Reply:
The [OpenSlide API](https://openslide.org/api/python/#openslide.OpenSlide.dimensions) handles a fixed number of dimensions (width, height) so we copy this over to `TileDBOpenSlide`. How to generalize this to more dimensions, or even if we should come up with a more generic API from scratch instead of trying to shoehorn it to OpenSlide is to be decided.